platform-sdk/platform-apps/demos/HelloSwirldDemo/src/main/java/com/swirlds/demo/hello/HelloSwirldDemoState.java

```diff
@@ -99,7 +99,7 @@ private HelloSwirldDemoState(final HelloSwirldDemoState sourceState) {
     }
 
     @Override
-    public synchronized void handleConsensusRound(final Round round, final PlatformState platformState) {
+    public synchronized void handleConsensusRound(final Round round, final PlatformStateAccessor platformState) {
```

Comment:
Why do we keep the `PlatformStateAccessor` around? Why not use the States API directly here? (I understand it's an override method, I'm asking about the base method and all the implementations.)
Or otherwise, why not use the Accessor everywhere instead of dealing with States API directly?

Reply:
The idea is that in some cases we may want to have an alternative and simple implementation of the platform state (`PlatformState`). When it comes to the services code, the strong preference is for States API. This API is more flexible but it's less straightforward than `PlatformStateAccessor`. So we keep it around for the platform code.